x-pack/plugins/viz_editor/public/common/components/operation_editor/operation_definitions.tsx

```diff
@@ -0,0 +1,621 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFieldNumber, EuiForm, EuiFormRow, EuiIcon, EuiRange, EuiSelect } from '@elastic/eui';
+import React from 'react';
+import {
+  AvgOperation,
+  CardinalityOperation,
+  ColumnOperation,
+  CountOperation,
+  DatasourceField,
+  DateHistogramOperation,
+  FieldOperation,
+  isFieldApplicableForScale,
+  SelectOperation,
+  SelectOperator,
+  SumOperation,
+  TermsOperation,
+  WindowFunction,
+  WindowOperation,
+} from '../../../../common';
+import { operationToName, VisModel } from '../../lib';
+
+const FixedEuiRange = EuiRange as React.ComponentType<any>;
+
+export type Scale = 'ordinal' | 'interval';
+export type Cardinality = 'single' | 'multi';
+
+export interface OperationEditorProps {
+  children: any;
+  column: SelectOperation;
+  onColumnChange: (newColumn: SelectOperation) => void;
+  onColumnRemove?: () => void;
```

Comment:
Please be consistent with the naming of this. This is the operation editor, so let's call the callbacks either:

```
  operation: SelectOperation;
  onOperationChange: (newOperation: SelectOperation) => void;
  onOperationRemove?: () => void;
```

or:

```
  operation: SelectOperation;
  onChange: (newOperation: SelectOperation) => void;
  onRemove?: () => void;
```

Reply:
I renamed a lot of columns in a lot of files - the only place left is the `Axis` interface but I want to clean this up separately because it has some other issues as well.